styles/style.css

```diff
@@ -246,6 +246,122 @@ h2 {
   margin-top: 24px;
   color: #bfbfbf;
 }
+.best-dest-heading {
+  margin-bottom: 104px;
+}
+.best-dest-cont {
+  padding: 120px 104px;
+}
+.best-dest-but-cont {
+  width: 50%;
```

Comment:
You shouldn't use width and height for this container, use for `.best-dest-but {min-width: 192px; padding: 17px 25px;}`
BTW it's such a nice name for button) maybe btn?